extensions/omni_httpd/master_worker.c

```diff
@@ -196,9 +196,19 @@ static void sigusr1_handler(int signo) {
  */
 void master_worker(Datum db_oid) {
   IsOmniHttpdWorker = true;
-  char socket_path_template[] = "omni_httpdXXXXXX";
-  char *tmpname = mkdtemp(socket_path_template);
-  socket_path = psprintf("%s/socket.%d", tmpname, getpid());
+  char *tmpdir = getenv("TMPDIR");
+  if (tmpdir == NULL || strcmp(tmpdir, "") == 0) {
+    // fallback to /tmp
```

Comment:
How about we try to see if there is a configuration variable... maybe `omni_httpd.temp_dir` that is set before falling back to /tmp?

Reply:
We can do that, I just followed the [POSIX](https://pubs.opengroup.org/onlinepubs/9699919799/xrat/V4_xbd_chap10.html)

> The /tmp directory is retained in POSIX.1-2017 to accommodate historical applications that assume its availability. Implementations are encouraged to provide suitable directory names in the environment variable TMPDIR and applications are encouraged to use the contents of TMPDIR for creating temporary files.

Reply:
even postgres test suite does the same thing
https://doxygen.postgresql.org/pg__regress_8c.html#ad9e2fb61733462b15438cffb898f4b56

Reply:
ok, let's just use `omni_httpd.temp_dir` with it's default value set to `/tmp`

Reply:
i'm using a custom guc now